_docs_integrate/request-one-time-consent-of-peer.md

```diff
@@ -25,88 +25,107 @@ required_by:
 # End automatic generation
 ---
 
-<!-- A general description of the requirement can be given here. -->
+This guide explains how an Identity can obtain the one-time consent of one of its peers on a particular issue using the [ConsentRequestItem]({% link _docs_integrate/data-model-overview.md %}#consentrequestitem). With the ConsentRequestItem it is possible to request the consent of a peer to an arbitrary text and thus reach agreement on a certain non-machine-processable context. The text for which the peer is asked for a one-time consent is specified in its `consent` property. To obtain a one-time consent, the Identity must send a [Request]({% link _docs_integrate/data-model-overview.md %}#request) to its peer that contains the corresponding ConsentRequestItem within its `items` property. The peer can accept or reject the ConsentRequestItem depending on whether or not the peer gives one-time consent to the text.
 
-With the ConsentRequest it is possible to request a consent of the peer to an arbitrary text and thus reach agreement on a certain non machine-processable context.
-
-To request an accept/reject decision from a peer to a free text, the ConsentRequestItem is used.
+Since understanding the process of asking a peer for a one-time consent requires knowledge about [Requests]({% link _docs_integrate/data-model-overview.md %}#request) in general, you should take a look at our [Request and Response introduction]({% link _docs_integrate/request-and-response-introduction.md %}) before continuing reading this guide. Further information on the [ConsentRequestItem]({% link _docs_integrate/request-and-response-introduction.md %}#consentrequestitem) can be found there as well.
+{: .notice--info}
 
 <!-- This include inserts the table with the metadata  -->
 
 {% include properties_list.html %}
 
-<!-- here is the description in detail  -->
+## Examples of consents {#consentrequestitem-examples}
 
-## Check the Request's validity
+There are many situations in which an Identity needs or wants the one-time consent of one of its peers.
+The corresponding text that the peer should agree to is contained within the `consent` property of a [ConsentRequestItem]({% link _docs_integrate/data-model-overview.md %}#consentrequestitem), for example:
 
-At first you should check if your Request is valid. You can do this by calling the `POST /api/v2/Requests/Outgoing/Validate` route on the Sender Connector with the following body.
-For the one-time consent request we use the [ConsentRequestItem]({% link _docs_integrate/data-model-overview.md %}#consentrequestitem).
-Even though the `peer` property is optional, it is recommended to specify it whenever possible. This allows additional validation rules to execute. When you are sending a Request via Messages you always know your peer.
+- "I hereby confirm the login to the website."
```

Comment:
Is this not an example for an AuthenticationRequestItem?

Reply:
I just left it as it was, unfortunately I do not know if this is a suitable example for the ConsentRequestItem. In any case, you are right that it is also listed as an example for the AuthenticationRequestItem. Any suggestions for a more appropriate example here?

Reply:
How about something like: "I hereby confirm that at the event pictures can be taken and published that may display me." Although I'm not certain, if you'd need a permanent consent in that case.

Reply:
Thanks! I had try to find a formulation that fits as an example for a one-time consent instead of a persistent consent.